SalesforceSDK/Core/Rest/ApiVersionStrings.cs

```diff
@@ -29,7 +29,7 @@ namespace Salesforce.SDK.Rest
 {
     public class ApiVersionStrings
     {
-        public const string VersionNumber = "v31.0";
+        public const string VersionNumber = "v35.0";
```

Comment:
@bhariharan Dont know about all prod instances, but http://na1.salesforce.com is on 35